### DISCUSSION TITLE
When there is an array with multiple billing_code_modifiers is that an AND or an OR?

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#736] [Q&A]
It would be helpful if the semantics were clear as to the meaning of an array with multiple billing_code_modifiers in the negotiated price object.  Is that meant as an AND of the modifiers (and will only match a claim line with all of those modifiers) or is the correct interpretation of multiple modifiers an OR?  

It seems like different payers are using this field in different ways and it would be very useful if the intention was clear in the spec.  

The other option would be to change the field to allow it to be used either an and AND or an OR, but indicate that in within the array.

Have others encountered this issue?

**Comment by medanalytics** [comment]
Could you share a screenshot of the sample data?

**Reply by nmayle** [reply]
I could but it would take time to get back to the original JSON.
If you have access to in-network data, just look for cases where there are two modifiers that are unlikely to come together e.g. LT and RT.  This is just an example.  

There are also payers that have arrays with ten or more modifiers on a single negotiated price object.  Clearly this isn't an AND; the standard claims forms only have room for up to four.  Most other payers seem to be using the billing_code_modifier field as an AND.

The language in the spec on GitHub doesn't make clear the semantics of this field when there are multiple modifiers in the array:

> An array of strings. There are certain billing code types that allow for modifiers (e.g. The CPT coding type allows for modifiers). If a negotiated rate for a billing code type is dependent on a modifier for the reported item or service, then an additional negotiated price object should be included to represent the difference. 

It would be great if the powers that be clarified the semantics of this field and put that clarification in an FAQ so the payers would know to check their code.

**Comment by tdfow** [comment]
Since the modifiers are an array within the negotiated price object data and the organization of the data is identifying unique pricing for a given service/provider, if the service with any of those modifiers result in the **same price**, same service code, same type, etc. then I feel the interpretation is any of those modifier values would be included in the billing_code_modifier array for the negotiated price object, so by default it seems to be an 'OR' situation.  
For example, if DME service E2603 has the same price with modifier RR (Rental) and modifier UE (purchase of used equipment), then seems appropriate to only publish once (OR condition) with both modifiers for the service's negotiated price object.  
If the NU modifier (new equipment) had a different price, then it would be noted in it's own negotiated price object.
 
<html>
<body>

billing_code_modifier | Billing Code Modifier | Array | An array of strings. There are certain billing code types that allow for modifiers (e.g. The CPT coding type allows for modifiers). If a negotiated rate for a billing code type is dependent on a modifier for the reported item or service, then an additional negotiated price object should be included to represent the difference.
-- | -- | -- | --


<br class="Apple-interchange-newline">
</body>
</html>

**Reply by nmayle** [reply]
@tdfow if the default interpretation is OR, then how would a payer ever represent AND?  Are there cases where the actual negotiated rate is dependent on two modifiers being applied to the code?  

'OR' can always be represented by multiple separate negotiated price objects, but if the default is 'OR', how would they represent 'AND' ?

**Reply by tdfow** [reply]
@nmayle  seems like someone from CMS would need to respond to your scenario.  
There are modifiers that don't impact rates of the service, so those modifiers wouldn't necessarily need to be reported unless it changes the rate or that would be how I interpreted the 'If a negotiated rate for a billing code type is dependent on a modifier for the reported item or service, then an additional negotiated price object should be included to show the difference."

**Reply by nmayle** [reply]
I believe there are cases when more than one modifier could apply and change the negotiated rate and in those cases, as on the line of a claim, it would be important to be clear that the interpretation is AND.  An example would be TC or 26 combined with other pricing modifiers.

The informational modifiers that don't change the rate would not necessarily be in the list reported by the payers. That seems to be a separate issue.
